models/modqueue.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+func (mql ModQueueListingChild) GetKind() string { return mql.Kind }
+func (mql ModQueueListingChild) GetId() string {
+	if mql.GetKind() == "t1" {
+		return mql.Data.(CommentListingDataChildrenData).Name
+	} else {
```

Comment:
no need to have an else here, the first return will already exit the scope

Reply:
I completely agree, thanks.